server/events.go

```diff
@@ -1613,7 +1613,7 @@ func (s *Server) accConnsUpdate(a *Account) {
 	if !s.eventsEnabled() || a == nil {
 		return
 	}
-	s.sendAccConnsUpdate(a, fmt.Sprintf(accConnsEventSubjOld, a.Name), fmt.Sprintf(accConnsEventSubjNew, a.Name))
+	s.sendAccConnsUpdate(a, fmt.Sprintf(accConnsEventSubjNew, a.Name))
```

Comment:
Why remove "old" subject? Would that possibly break apps listening to old subject? Don't mind dropping, but maybe not in 2.6.1...

Reply:
It would

Reply:
Possibly if apps were listening to those updates, but was messing up the accounting logic even beyond the counting JETSTREAM and ACCOUNT clients internally.

Reply:
We could send to both if you both feel strongly, but we can't process both as subs..

Reply:
I personally have no problem removing all together, but will trust your judgement here. But we can't process both for sure.

Reply:
we can still send them all we have to do is not subscribe in events.go line 780.

given this comment:
```
// accConnsUpdate is called whenever there is a change to the account's
// number of active connections, or during a heartbeat.
func (s *Server) accConnsUpdate(a *Account) {
```
how is processing the message twice different from a change in account followed by a heartbeat?